### ISSUE TITLE
TNT can only be activated by Flint and Steel

### ISSUE THREAD

**Issue by Spoonerj1** [GriefPrevention/GriefPrevention#1192]
Hey there, we are having trouble with tnt inside and outside claims. It appears that tnt can only be lit with flint and steel,  using any redstone mechanic will "light" the tnt, but no damage is done to blocks. 

this happens in claims, out of claims, and underground.

Is this expected behavior or a bug?

**Comment by RoboMWM** [comment]
Hmm, still not sure how people who already have accounts are getting by the template... is there some manual link out there that bypasses?

**Comment by Spoonerj1** [comment]
Hopefully this isnt something you actually need the template for. 

Expected behavior: vanilla ignition of tnt
Behavior seen: TNT only ignites with flint and steel. 

Tested on spigot with only grief protection installed. Verified by removing GP from the server and getting expected behavior. 

To replicate: attempt to light TNT with redstone components like a button or lever.

**Comment by RoboMWM** [comment]
config and version are things I never want to assume

**Comment by Spoonerj1** [comment]
You could find a much better way to communicate.

**Comment by Spoonerj1** [comment]
[06.01 11:53:03] [Server] [INFO] Server: Paper(345) 1.16.4-R0.1-SNAPSHOT
[06.01 11:55:43] [Server] [INFO] GriefPrevention version 16.16.0

[GPconfig.txt](https://github.com/TechFortress/GriefPrevention/files/5777637/GPconfig.txt)

**Comment by RoboMWM** [comment]
what do you mean a better way to communicate...?

anyways I'm assuming it's only occurring in that first world and none of the others? And does it change behavior if you disable any of the BlockExplosions configs?

**Comment by Spoonerj1** [comment]
No, the behavior did not change when I disabled those configs. 
It is occurring in only the world where claims are enabled (on the server this is "casimir") which was the same when I tested it with only the overworld, end, and nether with no other plugins.

**Comment by RoboMWM** [comment]
still wondering what you mean by better way to communicate.

Will check redstone tnt behavior I guess.
